Engine/ac/roomobject.cpp

```diff
@@ -149,12 +150,28 @@ void RoomObject::update_cycle_view_backwards()
       }
 }
 
-void RoomObject::ReadFromFile(Stream *in)
+void RoomObject::ReadFromFile(Stream *in, int32_t cmp_ver)
 {
     in->ReadArrayOfInt32(&x, 3);
     in->ReadArrayOfInt16(&tint_r, 15);
     in->ReadArrayOfInt8((int8_t*)&cycling, 4);
     in->ReadArrayOfInt16(&blocking_width, 2);
+    if (cmp_ver >= 1)
```

Comment:
Please give new save format a much higher number in case there will be change in 3.* branch